mhkit/river/io/d3d.py

```diff
@@ -1,127 +1,310 @@
-from os.path import abspath, dirname, join, isfile, normpath, relpath
 from mhkit.utils import unorm
-from math import isclose
 import scipy.interpolate as interp
-import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
 import netCDF4
+import warnings
 
 
-def get_layer_data(data, variable, layer_index= -1 , time_index=-1):
+def get_all_time(data):
     '''
-    Get variable data from netcdf4 object at specified layer and timestep. 
+    Returns all of the data time stamps in seconds.
```

Comment:
I think rewording this could clarify the meaning... maybe something like:
> Returns all of the time stamps from a D3D simulation passed to the function as a NetCDF object (`data`)